frontend/e2e/tests/global-teardown.ts

```
@@ -0,0 +1,14 @@
+import { test as teardown } from '@playwright/test'
+
+/**
+ * Global teardown - run once after all tests
+ * Clean up test data if needed
+ */
+teardown('cleanup', async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)